distribution/src/resources/drill-override-example.conf

```diff
@@ -184,7 +184,13 @@ drill.exec: {
       # Set this property if custom absolute root should be used for remote directories
       root: "/app/drill"
     }
-  }
+  },
+  # Settings for Temporary Tables.
+  # See https://gist.github.com/arina-ielchiieva/50158175867a18eee964b5ba36455fbf#file-temporarytablessupport-md.
```

Comment:
In stead of pointing to a gist file in Drill source code, can we point to DRILL-4956 which points to such document? After Drill doc updates with the temporary table, we can update the link in DRILL-4956.  That way, we do not have to update the code to change the link later on.